wiki/Chat_Console/tr.md

```diff
@@ -0,0 +1,107 @@
+| **Pembe** | [BanchoBot](/wiki/BanchoBot) |
+
+- Sohbet konsolu kapalıyken en güncel mesajların ekranın en altında gözükmesi için `Show Ticker` kutucuğunu işaretleyin.
+- Sohbeti oynanış sırasında (beatmapin introsu, outrosu ve aralar hariç) otomatik olarak gizlemek için `Auto-Hide` kutucuğunu işaretleyin.
+- Sohbet konsolunu gizlemek için `Hide Chat` kutucuğuna tıklayın veya `F8` tuşuna tekrar basın.
+
+## Genişletilmiş Sohbet Konsolu
+
+*[osu!academy](/wiki/osu!academy) bu sayfayı [Çoklu](/wiki/Multi) ile birlikte [Episode 6(6:52)](https://www.youtube.com/watch?v=cyYRl-a5xII) videosunda ele aldı.*
+
+osu!'daki pek çok ekranda `F9` tuşuna ya da Intro Ekranı'nın sağ altındaki `Online Users` butonuna basarak Genişletilmiş Sobhet Konsolu'na erişebilirsiniz. Genişletilmiş Sohbet, Sohbet Konsoluna ek olarak ekranın geri kalan üçte ikisini kaplayarak şu anda osu'da oturum açmış olan kullanıcılar hakkında bilgi verir!
+
+![Extended chat console](img/Chat_Console-Extended.png "Genişletilmiş sohbet konsolu")
+
+osu!'da oturum açmış olan her bir kullanıcının Genişletilmiş Sohbet'te görüntülenen bir kullanıcı paneli vardır. Varsayılan olarak, genel bilgileri (ad, toplam sıralama puanı, sıralama, doğruluk, oynatma sayısı ve varsa kullanıcının avatarı) görüntüler. Fare imleci üzerine yerleştirildiğinde, farklı bir bilgi kümesi (ad, sıralama, varsa avatar, yerel saat, saat dilimi, ülkeye göre konum ve kullanıcı izin verdiyse, şehir, ve ne yaptıklarını) görüntüler.
+
+- Arkadaşlar sadece arkadaşlarınızın ekranda görünmesini sağlar.
+- Lock Panels panellerin etrafta sekip durmasını ve yeni kullanıcıların ekrana düşmesini engeller.
+- Kullanıcı panellerini bir özniteliğe göre sıralamak için o sekmeye tıklayın.
+- Herkesin nerede olduğunu gösteren bir dünya haritasını görüntülemek için dünya haritasına tıklayın.
+- Kaydırmak için beyaz kutuyu tıklatıp sürükleyebilirsiniz, fare tekerleğini de kullanabilirsiniz.
+- Panellerinde istatistik olmayan kullanıcılar IRC istemcilerini kullanarak sohbete bağlanmıştır.
+
+| Panel rengi | Açıklama |
+| :-- | :-- |
+| ![Player is idle](img/Chat_Console-Idle.png "Oyuncu boşta") | Lacivert - Oyuncu şu anda boşta, bir şey yapmıyor ya da sadece sohbet ediyor. |
+| ![Player is playing](img/Chat_Console-Playing.png "Oyuncu oyunda") | Gri - Solo'da bir beatmap oynuyor. |
+| ![Player is watching](img/Chat_Console-Watching.png "Oyuncu izliyor") | Açık Mavi - Bir tekrarı veya birini izliyor. |
+| ![Player is editing a beatmap](img/Chat_Console-Editing.png "Oyuncu bir beatmap düzenliyor") | Kırmızı - Kendine ait bir beatmapi düzenliyor. |
+| ![Player is testing a beatmap](img/Chat_Console-Testing.png "Oyuncu bir beatmap test ediyor") | Mor - Kendisinin ya da bir başkasının beatmapini test ediyor. |
+| ![Player is submitting a beatmap](img/Chat_Console-Submitting.png "Oyuncu bir beatmap gönderiyor") | Turkuaz - Yaptıkları bir beatmapi gönderiyor (yükleme veya güncelleme). |
```

Comment:
```suggestion
| ![Player is submitting a beatmap](img/Chat_Console-Submitting.png "Oyuncu bir beatmap gönderiyor") | Turkuaz - Yaptığı bir beatmapi gönderiyor (karşıya yükleme veya güncelleme). |
```
thought this would look better